test/unit/sql/SqlService.js

```diff
@@ -412,76 +414,6 @@ describe('SqlServiceTest', function () {
             ).should.be.true;
         });
     });
-    describe('handleExecuteResponse', function () {
```

Comment:
why are these tests deleted?

Reply:
Also faking invocation service made some tests fail because as you know we reject or resolve invocations inside invocation service after calling the handler. If we don't mock the service, the client will try to send invocations. So I will delete some tests.